scripts/make_next_milestone.py

```diff
@@ -47,7 +47,10 @@ def main():
         desc = m["description"]
         res = re.search(RELEASE_MANAGER_RE, desc)
         if res is not None:
-            maintainers.remove(res.group(1))
+            try:
+                maintainers.remove(res.group(1))
```

Comment:
Oh, it was rather unexpected for me
I thought about such cases since they are possible (see 123 and 124 releases) but I was completely sure that `remove` in such cases just did nothing like in Java
So, nice catch! 👍